agent/node.go

```diff
@@ -607,8 +607,9 @@ func (n *Node) runManager(ctx context.Context, securityConfig *ca.SecurityConfig
 			return err
 		}
 		done := make(chan struct{})
+		var runErr error
 		go func() {
-			m.Run(context.Background()) // todo: store error
+			runErr = m.Run(context.Background())
```

Comment:
I wonder if it would be a little cleaner to change done to `runErr chan error` and end the error over the channel.